lib/sender.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const net = require('net')
+const tls = require('tls')
+const { Writable } = require('stream')
+
+
+/**
+ * Event sender.
+ * Can be used as a stream, with the usual functions:
+ * sender.write(), sender.end(), etc.
+ * Implements backpressure.
+ */
+class Sender extends Writable {
+
+  /**
+   * Create the client.
+   * @param {Object} options for sender, including:
+   *  - host: the host to send events, default localhost.
+   *  - port: the destination port (mandatory).
+   *  - objectMode: whether to allow sending objects.
+   * @param {function(*)} callback (Optional) Function to invoke when the
+   * connection has been established.
+   */
+  constructor(options, callback) {
+    super(options)
+    this._options = options
+    const libnet = options.cert ? tls : net
+    this._socket = libnet.connect(options, callback)
+    this._socket.on('error', error => this.emit('error', error))
+    this._socket.unref()
```

Comment:
Why do you want to change the default behaviour?

Reply:
You are right, I changed it a little while back.

Reply:
and isn't it better to delete the line?